### PR TITLE
readme update for llama-405B

### PR DESCRIPTION
**Pull request by hsubramony**
# What does this PR do?

Fixes # (issue)

## Before submitting
- [ ] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [ ] Did you make sure to update the documentation with your changes?
- [ ] Did you write any new necessary tests?

### CODE REVIEW

File: examples/text-generation/README.md [hunk 1]
```diff
@@ -187,6 +187,20 @@ python ../gaudi_spawn.py --use_deepspeed --world_size 8 run_generation.py \
 --flash_attention_causal_mask
 ```
 
+To run Llama-405B inference on 8 Gaudi3 cards use the following command:
+```bash
+python ../gaudi_spawn.py --use_deepspeed --world_size 8 run_generation.py \
+--model_name_or_path llama3.1/Meta-Llama-3.1-405B-Instruct \
```
Comment (shepark) [inline examples/text-generation/README.md:193]: Why don't you change this to official name?
```suggestion
--model_name_or_path meta-llama/Llama-3.1-405B-Instruct \
```

File: examples/text-generation/README.md [hunk 2]
```diff
@@ -338,7 +352,7 @@ PT_ENABLE_INT64_SUPPORT=1 PT_HPU_LAZY_MODE=0 python ../gaudi_spawn.py  --world_s
...
+Llama2-70b, Llama2-7b, Llama3-70b, Llama3-8b, Mixtral-8x7B, Falcon-7B, Falcon-40B, Falcon-180B, phi-2 and Llama-405B in FP8 are enabled using the [Intel Neural Compressor (INC)](https://docs.habana.ai/en/latest/PyTorch/Inference_on_PyTorch/Inference_Using_FP8.html), which provides model measurement and quantization capabilities in PyTorch. From synapse 1.17 / optimum-habana 1.13 release, INC is used by default for measuring and quantization. Habana Quantization Toolkit (HQT), which was used earlier, will be removed in future releases. To use HQT, disable INC by setting the following environment variable: `USE_INC=0`.
```
Comment (shepark) [inline examples/text-generation/README.md:355]: and move location and same pattern as others
```suggestion
Llama3-8b, Llama3-405B, Mixtral-8x7B
```

File: examples/text-generation/README.md [hunk 3]
```diff
@@ -461,6 +475,44 @@ QUANT_CONFIG=./quantization_config/maxabs_quant.json python ../gaudi_spawn.py \
 --flash_attention_causal_mask
 ```
 
+Here is an example to measure the tensor quantization statistics on Llama-405B with 8 cards:
+> Please note that Llama-405B requires minimum 16 cards Gaudi2 and 8 cards Gaudi3.
+```bash
+QUANT_CONFIG=./quantization_config/maxabs_measure_include_outputs.json python ../gaudi_spawn.py \
+--use_deepspeed --world_size 8 run_lm_eval.py \
+-o acc_falcon180b_bs1_quant.txt \
```
Comment (shepark) [inline examples/text-generation/README.md:483]:
```suggestion
-o acc_llama3_405b_bs1_quant.txt \
```

File: examples/text-generation/README.md [hunk 4]
```diff
@@ -187,6 +187,20 @@ python ../gaudi_spawn.py --use_deepspeed --world_size 8 run_generation.py \
 --flash_attention_causal_mask
 ```
 
+To run Llama-405B inference on 8 Gaudi3 cards use the following command:
```
Comment (shepark) [inline examples/text-generation/README.md:190]: Why don't we have same pattern as other Llama models in this document?
```suggestion
To run Llama3-405B inference on 8 Gaudi3 cards use the following command:
```

File: examples/text-generation/README.md [hunk 5]
```diff
@@ -461,6 +475,44 @@ QUANT_CONFIG=./quantization_config/maxabs_quant.json python ../gaudi_spawn.py \
 --flash_attention_causal_mask
 ```
 
+Here is an example to measure the tensor quantization statistics on Llama-405B with 8 cards:
+> Please note that Llama-405B requires minimum 16 cards Gaudi2 and 8 cards Gaudi3.
+```bash
+QUANT_CONFIG=./quantization_config/maxabs_measure_include_outputs.json python ../gaudi_spawn.py \
+--use_deepspeed --world_size 8 run_lm_eval.py \
+-o acc_falcon180b_bs1_quant.txt \
+--model_name_or_path llama3.1/Meta-Llama-3.1-405B-Instruct \
```
Comment (regisss) [inline examples/text-generation/README.md:484]:
```suggestion
--model_name_or_path meta-llama/Llama-3.1-405B-Instruct \
```

File: examples/text-generation/README.md [hunk 6]
```diff
@@ -461,6 +475,44 @@ QUANT_CONFIG=./quantization_config/maxabs_quant.json python ../gaudi_spawn.py \
...
+--flash_attention_recompute \
+--flash_attention_causal_mask
+```
+
+Here is an example to quantize the model based on previous measurements for Llama-405B with 8 cards:
+> Please note that Llama-405B requires minimum 16 cards Gaudi2 and 8 cards Gaudi3.
+```bash
+QUANT_CONFIG=./quantization_config/maxabs_quant.json python ../gaudi_spawn.py \
+--use_deepspeed --world_size 8 run_generation.py \
+--model_name_or_path llama3.1/Meta-Llama-3.1-405B-Instruct \
```
Comment (regisss) [inline examples/text-generation/README.md:501]:
```suggestion
--model_name_or_path meta-llama/Llama-3.1-405B-Instruct \
```

**Review by regisss — APPROVED** [review]

**Pull request merged**